src/utils.ts

```diff
@@ -2224,12 +2245,10 @@ export class newAxis {
     return this.isVertical ? value * this.transformMatrix.d + this.transformMatrix.f : value * this.transformMatrix.a + this.transformMatrix.e
   }
 
-  public normedValue(value: number): number {
-    return value / this.interval
-  }
+  public normedValue(value: number): number { return value / this.interval }
 
   private computeMinMax(vector: any[]): number[] {
-    if (this.isDiscrete) { return [0, this.labels.length] };
+    if (this.isDiscrete) { return [0, this.labels.length - 1] };
```

Comment:
If you fancy it, you can remove redundant brackets here (`if (this.isDiscrete) return [0, this.labels.length - 1];`). That is cleaner for one-liner ifs

Reply:
Done